tf2_py/src/tf2_py.cpp

```diff
@@ -64,11 +64,11 @@ static PyTypeObject buffer_core_Type = {
 # else
   PyVarObject_HEAD_INIT(NULL, 0)
 #endif
-  "_tf2.BufferCore",               /*name*/
-  sizeof(buffer_core_t),           /*basicsize*/
+  "_tf2.BufferCore",         /* name */
+  sizeof(buffer_core_t),     /* basicsize */
```

Comment:
There's actually still a bug here. That structure isn't initialized completely. I pasted in  (and aligned comments to) what I thought was the full initialization from http://python3porting.com/cextensions.html , but it still showed a lot of warnings and seemed incomplete.